Validation/HcalHits/plugins/HcalSimHitStudy.cc

```diff
@@ -512,6 +515,7 @@ void HcalSimHitStudy::analyzeHits(std::vector<PCaloHit> &hits) {
 
   for (int i = 0; i < ieta_bins_HF; i++) {
     for (int j = 0; j < iphi_bins; j++) {
+      [[clang::suppress]]
```

Comment:
looks like this is false positive. All these are properly initialized in https://github.com/cms-sw/cmssw/blob/master/Validation/HcalHits/plugins/HcalSimHitStudy.cc#L296-L318